integration/test/dashboard_json_test.go

```diff
@@ -49,6 +57,14 @@ var excludeCounters = []string{
 	"cluster_peer",
 	"efficiency_savings",
 	"ems_events",
+	"external_service_op_num_not_found_responses",
```

Comment:
could simplify to `external_service_op_num_` and `external_service_op_request_` unless that excludes things that should be checked

Reply:
Done